helmfile/charts/prometheus-alerts/templates/rules/predict-linear.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.predictLinear}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "predict-linear" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+{{ include "prometheus-alerts.labels" . | indent 4 }}
+{{- if .Values.defaultRules.labels }}
+{{ toYaml .Values.defaultRules.labels | indent 4 }}
+{{- end }}
+{{- if .Values.defaultRules.annotations }}
+  annotations:
+{{ toYaml .Values.defaultRules.annotations | indent 4 }}
+{{- end }}
+spec:
+  groups:
+  - name: predict-linear
+    rules:
+    - alert: PersistentVolumeSixtySixProcentInThreeDays
+      annotations:
+        message: The PersistentVolume claimed by {{`{{ $labels.persistentvolumeclaim }}`}} in Namespace {{`{{ $labels.namespace }}`}} is full in three days.
+      expr: predict_linear(kubelet_volume_stats_available_bytes[24h], 3*24*60*60) <= (kubelet_volume_stats_capacity_bytes*(1-{{ .Values.predictLinearLimit }}))
+      for: 5m
+      labels:
+        severity: warning
+    - alert: MemorySixtySixProcentInThreeDays
+      annotations:
+        message: Memory usage in Cluster {{`{{ $labels.cluster }}`}} Instance {{`{{ $labels.instance }}`}} is predicted to go over sixty six percent within the next 3 days at current use rate.
+      expr: predict_linear(node_memory_MemAvailable_bytes[24h], 3*24*60*60) <= (node_memory_MemTotal_bytes*(1-{{ .Values.predictLinearLimit }}))
```

Comment:
Since we have a configurable limit it would be nice if the message also used that limit instead of always saying "66 percent". Also the name could be something more generic that does not mention "66 percent".

Same thing for the other alerts.

Reply:
Done!